app/preferences_wasm.go

```diff
@@ -0,0 +1,62 @@
+//go:build wasm
+
+package app
+
+import (
+	"bytes"
+	"io"
+	"strings"
+	"syscall/js"
+)
+
+const preferencesLocalStorageKey = "preferences.json"
```

Comment:
possibly "fyne/preferences.json" or "Fyne-preferences.json" - as you say to reduce the possibility of overlap.

Reply:
Done